src/components/Tabs/TabPanel.tsx

```diff
@@ -1,6 +1,7 @@
-import { TabPanel as ReachTabPanel, TabPanelProps as ReachTabPanelProps } from '@reach/tabs';
+import React from 'react';
+import { TabPanel as ReachTabPanel } from '@reach/tabs';
+import Box from '../Box';
 
-export type TabPanelProps = ReachTabPanelProps;
-export const TabPanel = ReachTabPanel;
+const TabPanel: React.FC = props => <Box as={ReachTabPanel} tabIndex={-1} {...props} />;
```

Comment:
That's breaking a screen reader as we need to ensure that the content is stacked properly.

Reply:
how can I tell the screenreader "don't focus on the tabpanel, but on its content"?

The reason this focus exissts is because the lib that we are using had a `tabIndex=1` on a `div`